server/src/configs/app.js

```diff
@@ -0,0 +1,39 @@
+
+import bodyParser from 'body-parser';
+import routes from '../routes';
+import express from 'express';
+
+const app = (
+
```

Comment:
Is this line feed needed?

Reply:
It would be better if we pass some variables in the future and someone else is passing variables. It would become easy to resolve the conflict in that case.